### DISCUSSION TITLE
Add flag support when dumping schema

### DISCUSSION THREAD

**Discussion by nkelton** [amacneil/dbmate#547] [Feature Ideas]
`pg_dump` in Postgresql supports [multiple flag arguments](https://www.postgresql.org/docs/current/app-pgdump.html) when dumping the schema and it would be nice if they were supported through the dbmate cli.  This is particularly useful when you want to dump the schema without populating all the database data

**Comment by amacneil** [comment]
>  This is particularly useful when you want to dump the schema without populating all the database data

We don't populate the database data.

**Comment by dossy** [comment]
I think adding the ability to specify additional command line arguments to whatever dump command dbmate is using for the data source is a needed thing, as brought up by discussion #625, where the desire is to invoke `SET ROLE` upon connection, and this would also require being able to specify `--role <rolename>` to the `pg_dump` command, as well.

**Comment by dossy** [comment]
Duplicates #310.
